docs/api-docs/channels/channels-storefront-tutorial.md

```diff
@@ -143,55 +109,31 @@ The following protected sections are provided by BigCommerce.
 | `Notifications`       | `notifications`       | Renders channel specific notification settings |
 | [`Currencies`](#currencies-settings)          | `currencies`          | Renders channel specific currency settings     |
 
-Include protected sections in the [create channel request](/api-reference/store-management/channels/channels/createchannel) to display BigCommerce provided channel specific settings.
+Include protected sections in the [create channel menus request](/api-reference/store-management/channels/channels/postChannelMenus) to display BigCommerce-provided channel-specific settings.
 
 ```http
-POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/channels
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/channels/{{CHANNEL_ID}}/channel-menus
 X-Auth-Token: {{ACCESS_TOKEN}}
 Content-Type: application/json
 Accept: application/json
 
 {
-  "name": "Dog US",
-  ...
-  "config_meta": {
-    "app": {
-      "id": 112233,
-      "sections": [
-        {
-          "title": "Storefront Settings",
-          "query_path": "storefront_settings"
-        },
-        {
-          "title": "Domains",
-          "query_path": "domains"
-        },
-        {
-          "title": "Notifications",
-          "query_path": "notifications"
-        },
-        {
-          "title": "Currencies",
-          "query_path": "currencies"
-        }
-      ]
-    }
-  }
+  "bigcommerce_protected_app_sections": [
+    "storefront_settings",
+    "domains",
+    "notifications",
+    "currencies"
+  ]
 }
 ```
 
 Included protected sections display above custom sections.
 
 ![Protected Sections](https://storage.googleapis.com/bigcommerce-production-dev-center/images/channels/channels-sf-protected-custom-settings.png "Protected Sections")

```

Comment:
Is this how this actually looks (with the line separating protected sections from custom)? I don't think so, so we need a different image. I would also recommend the image including the URL so people can see how the URL resolves for the protected sections

Reply:
@amckemie Just tested this out and it does have a horizontal line separating the two 
![image](https://user-images.githubusercontent.com/29524703/171681241-e3809568-7355-4aaf-96a8-902c339fa735.png)